ups/eupspkg.cfg.sh

```diff
@@ -1,6 +1,6 @@
 # EupsPkg config file. Sourced by 'eupspkg'
 
-MINICONDA2_VERSION=${MINICONDA2_VERSION:-3.19.0} # Version of Miniconda to install
+MINICONDA3_VERSION=${MINICONDA3_VERSION:-4.0.5} # Version of Miniconda to install
```

Comment:
That was current when I wrote the code. It's taken a very long time for people to review it. You will note that our miniconda2 is out of date as well. I think I'll leave it like that because I don't want to have to redo all the version package lists as well. We should start moving 2 and 3 together as part of release management.